packages/ui-kit/src/icons/lord-icon/building.json

```diff
@@ -2047,7 +2047,7 @@
               "nm": "Color",
               "mn": "ADBE Color Control-0001",
               "ix": 1,
-              "v": { "a": 0, "k": [0.027, 0.549, 0.875], "ix": 1 }
```

Comment:
@timonrey please ensure all the .json have the right color (I might be wrong but it looks some are missing). Also all the `../svg/lord-icon-*` need to change their color. These are the static assets used in the SSR version of the page before hydration

Reply:
@gabriele-ct I changed the svg's now too, but only the ones that we currently use in the mega menu. There are 6 other lordicons that were not in the design folder and are also not used in the mega menu atm. Do we need them for other purposes?

Reply:
@timonrey only the ones that are used in the mega menu needs to change color, the others are typically more complex svgs that are used in other parts of the site (mostly on the learning path homepage)